src/test/java/org/folio/rest/impl/VouchersApiTest.java

```diff
@@ -73,4 +79,18 @@ public void testGetVouchersVoucherByIdInvalidFormat() {
     assertTrue(actual.contains(INVALID_VOUCHER_ID));
   }
 
+  @Test
+  public void testGetVoucherNumberStart() {
+    logger.info("=== Test Get Voucher number start value - 500 Internal Server Error ===");
```

Comment:
Log message should be for 200 and not 500

Reply:
Thanks, corrected